pkg/storaged/btrfs/utils.jsx

```diff
@@ -75,3 +79,12 @@ export function parse_subvol_from_options(options) {
     else
         return null;
 }
+
+export function validate_subvolume_name(name) {
+    if (name === "")
+        return _("Name cannot be empty.");
+    if (name.length > 255)
+        return _("Name cannot be longer than 255 characters.");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19849-20240117-090134-2dfecb5e-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)